src/types/user.interface.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
**thought(non-blocking):** Users has a lot of fields that were inherited from an earlier attempt at a core API. I think we should focus on only interfacing the things Kanvas needs to operate properly. What do you think? 💭

Reply:
Agree, lets discuss the fields that we are going to keep on User interface